appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/admin/ManagedExecutorServiceBaseManager.java

```diff
@@ -170,6 +172,7 @@ protected void setAttributes(HashMap attributes, String target) {
         contextInfoEnabled = (String) attributes.get(CONTEXT_INFO_ENABLED);
         threadPriority = (String) attributes.get(THREAD_PRIORITY);
         longRunningTasks = (String) attributes.get(LONG_RUNNING_TASKS);
+        useForkJoinPool =  (String) attributes.get(USE_FORK_JOIN_POOL);
```

Comment:
Knock-on from suggestion to move `ManagedExecutorService` specific config out of `ManagedExecutorServiceBase` interface, this would need adjusting.